README.md

```diff
@@ -47,6 +48,20 @@ The Flexy's clock must be set to Coordinated Universal Time (UTC) in order to po
 
 Using FTP transfer the contents of the osisoft_connector directory to the /usr/ directory of your eWON via FTP.  If successful you should have a css directory, js directory, config.html, ConnectorConfig.json, flexy-osisoft-connector.jar, and jvmrun located in the /usr/ directory of the Flexy.  You must also follow all steps in the Certificates section below.
 
+### Tag's Configuration
+
+The tag's historical logs are the source for the data pushed to OSIsoft, so settings for your historical logging will affect what data can get sent to OSIsoft. Make sure you enable historical logging in your tag settings. 
++ On the flexy web page, navigate to Tags -> values
++ Via the button on the top left section of the web page, switch the mode from view to setup
```

Comment:
Maybe a picture like [hms-networks/IgnitionEwonConnector](https://github.com/hms-networks/IgnitionEwonConnector) has would be better. It just seems very vague without more detail. There's multiple buttons in that section